integrations/server/test_covidcast_meta.py

```diff
@@ -10,6 +10,7 @@
 #first party
 from delphi_utils import Nans
 from delphi.epidata.acquisition.covidcast.covidcast_meta_cache_updater import main as update_cache
+import delphi.operations.secrets as secrets
```

Comment:
just want to point out that nothing in these tests is actually testing that the meta values were computed correctly, only that the length of the responses returned was correct and that fields like `min_time` are present. i don't believe that the changes to `compute_covidcast_meta` warrant and update to that, but it would be good to have at least a few tests that compare hard values in the future.

Reply:
+1, would you create an Issue for that?

Reply:
#924